src/transformers/generation/utils.py

```diff
@@ -2514,7 +2519,7 @@ def _greedy_search(
         batch_size = input_ids.shape[0]
         this_peer_finished = False
         unfinished_sequences = torch.ones(batch_size, dtype=torch.long, device=input_ids.device)
-        model_kwargs = self._get_initial_cache_position(input_ids, model_kwargs)
+        #model_kwargs = self._get_initial_cache_position(input_ids, model_kwargs)
```

Comment:
is this change needed?

Reply:
yeah this is a weird bug causing fail in normal cache, even if it wasn't touched. Will be removed soon :)